src/main/java/duke/FileHandler.java

```diff
@@ -0,0 +1,143 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileNotFoundException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
```

Comment:
I like how you separated all the different imports according to how and when they were used.